src/afcharts/pio_template.py

```diff
@@ -1,152 +1,155 @@
+# instead.
+half_line = base_size / 2
+
+pio.templates["theme_af"] = go.layout.Template(
+    layout={
+        "autosize": True,  # Automatically adjusts the scale of the plot based on it's content
+        "annotationdefaults": {
+            "font": {"size": base_size},
+            "showarrow": False,
+        },  # Sets default font size for annotation
+        "bargap": 0.15,
+        "bargroupgap": 0.1,
+        "coloraxis": {
+            "colorbar": {  # Bar chart colours
+                "outlinewidth": 0,  # Width of the outline around the color bar
+                "tickcolor": af_colour_values["chart_features"],  # Bar chart tick colour
+                "ticklen": half_line / 2,  # Bar chart tick length
+                "ticks": "outside",  # Bar chart tick position
+            }
+        },
+        "colorscale": {
+            "sequential": sequential,  # Sequential colour scale for low to high ranges
+            "sequentialminus": sequential_minus,  # Sequential palette for negatives
+            "diverging": diverging,  # Diverging colour scale
+        },
+        "colorway": main,  # Sequence of colours to be used in plots
+        "font": {
+            "color": "black",  # Text colour
+            "family": afcharts_font,  # Font
+            "size": base_size,
+        },  # Text size
+        "legend_title": None,  # Removes legend title
+        "legend": {
+            "borderwidth": 0,
+            "title": {"text": None},  # Removes legend title
+            "font": {"size": base_size},  # Legend font size
+            "bgcolor": "rgba(0,0,0,0)",  # Makes legend background transparent
+            "orientation": "v",  # Legend orientation
+            "x": 1,  # Positions legend (0,0 is the bottom left)
+            "y": 0.5,
+            "indentation": 0,
+            "itemclick": "toggleothers",  # Change behaviour from hiding trace to showing only this trace
+            "itemwidth": 30,
+            "traceorder": "normal",
+        },
+        "hoverlabel": {
+            "align": "left",  # Align hover label text to the left
+            "font_size": base_size * 0.9,  # Text size of hover
+            "bgcolor": "white",  # Hover box background
+        },
+        "hovermode": "x unified",  # How hovering affects the display
+        # x unified shows info for all the data at that point in the x-axis
+        "paper_bgcolor": "rgba(0,0,0,0)",  # Makes paper (entire area) background transparent
+        "plot_bgcolor": "rgba(0,0,0,0)",  # Makes plot area transparent
+        "margin": {  # Set margins around the plot area in pixels
+            "l": half_line,  # Left margin
+            "r": half_line,  # Right margin
+            "t": (base_size * 1.6) + (0.7 * (base_size * 1.6)) + half_line + base_size,
+            # Top margin is the size of the title + subtitle size + title_padding
+            # + space between title and subtitle (which is set by default)
```

Comment:
As noted above, title, subtitle and source shouldn't be embedded in the chart anyway so we don't need to make allowances for these in the margins.

```suggestion
            "t": half_line,  # Top margin
```